ISSUES.md

```diff
@@ -0,0 +1,57 @@
+# Issues
+
+## How to create new issues properly
+
+GitHub issues are used for everything from bug reporting to long-term ideas. As such you can make everything much smoother by following some simple rules.
+
+### Rule zero
+
+Always try to give your issue a meaningful title as this is the first thing anyone will see.
+
+Note: `[CR]` and `[WIP]` "tags" are meaningful only for PRs. All open issues by definition are request for comments and work in progress.
+
+### Bug reports
+
+Before you submit a bug always search the issues to see if it hasn't been reported already.
+
+Your bug report has to include:
+
+- On what OS did you experience the problem (Windows, Linux, OS X etc.)
+- What version were you playing:
+ - Tiles or Curses (text-based)
+ - Version string (preferably full version e.g. "0.C-4547-g3f1c109", or Jenkins build number e.g. 3245)
```

Comment:
I would rather not suggest a build number, people seem to latch onto that and use it to the exclusion of a proper version number, and it's a pain in the ass to have to look it up on jenkins just to find out what version they're talking about.